src/test/DittoMachine.t.sol

```diff
@@ -17,7 +17,7 @@ contract ContractTest is TestBase {
         string memory uri721 = nft.tokenURI(nftId721);
         currency.mint(address(this), MIN_AMOUNT_FOR_NEW_CLONE);
         currency.approve(dmAddr, MIN_AMOUNT_FOR_NEW_CLONE);
-        uint256 clone0Id = dm.duplicate(nftAddr, nftId721, currencyAddr, MIN_AMOUNT_FOR_NEW_CLONE, false);
+        (uint256 clone0Id, , ) = dm.duplicate(nftAddr, nftId721, currencyAddr, MIN_AMOUNT_FOR_NEW_CLONE, false, 0);
```

Comment:
I noticed we are using different naming style in the file - `cloneId1` and `clone1Id`. Would be good to standardise it.